### PR TITLE
Add FAQ

### PR DESCRIPTION
**Pull request by Duncaen**
Initial effort to get a nice FAQ on the voidlinux site, nicely formatted with up to date and correct information.
There is still a lot todo, but I guess its good to publish it already to collect criticism, help and suggestions.
This is loosely inspired by the OpenBSD FAQ.

### CODE REVIEW

**Comment by Gottox** [comment]
There's already an FAQ on the wiki. Do you think they could conflict?

File: faq/index.markdown [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+
+# Installation
+
+Void Linux provides a dialog based installation script, which can be used for most basic setups.
+The script however does not work for slightly more specific setups, in this case you should use the manual installation method.
+
+The following features are **not** supported by the installation script:
+
+* LVM
+* Luks
```
Comment (Vaelatern) [inline faq/index.markdown]: `* ZFS`

File: faq/index.markdown [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+#!/bin/sh
+ssh-keygen -A >/dev/null 2>&1 # Will generate host keys if they don't already exist
+[ -r conf ] && . ./conf
+exec /usr/bin/sshd -D $OPTS
+# echo 'OPTS="-p 2222"' >>/etc/sv/sshd/conf
+```
+
+### Cron
+
+Void Linux comes without a default cron daemon, you can choose one of multiple cron implementations, by installing the package and enabling the system service.
```
Comment (Vaelatern) [inline faq/index.markdown]: Mention how any cron agent can be enabled just by enabling the one symlink (via alternatives)?

File: faq/index.markdown [hunk 3]
```diff
@@ -0,0 +1,390 @@
...
+
+## Changing the default shell
+
+The default shell for users can be changed with the `chsh(1)` tool.
+
+```
+$ chsh -s /bin/bash <user name>
+```
+
+A list of available shells can be retrieved with the `chsh(1)` list command.
```
Comment (Vaelatern) [inline faq/index.markdown]: `that you have already installed`

**Comment by Vaelatern** [comment]
I don't think multiple FAQs would conflict. Like an A/B test, one would eventually win.

**Comment by Duncaen** [comment]
This "FAQ" is more like a guide or official manual like the OpenBSD "FAQ", less like a usual FAQ with just question answer.
https://www.openbsd.org/faq/

Maybe we should rename it to "Guide", "Manual" or "Handbook".

File: faq/index.markdown [hunk 4]
```diff
@@ -98,6 +98,62 @@ The installer has to be executed as `root` user, if you logged in as `anon` you
...
+
+Some applications require PulseAudio, especially closed source programs.
+
+#### Alsa
+
+Install the `alsa-utils` package make sure your user is part of the `audio` group to access audio devices.
+
+```
+# xbps-install -S alsa-utils
+# usermod -a -G <username> audio
```
Comment (chneukirchen) [inline faq/index.markdown]: wrong argument order, no?

Comment (Gottox) [inline faq/index.markdown]: Yea, audio <username>

File: faq/index.markdown [hunk 5]
```diff
@@ -98,6 +98,62 @@ The installer has to be executed as `root` user, if you logged in as `anon` you
...
+Alternatively using the alsa configuration file `/etc/asound.conf` or the per-user configuration file `~/.asoundrc` to set a different card as the default.
+
+```
+defaults.ctl.card 2;
+defaults.pcm.card 2;
+```
+
+#### PulseAudio
+
+PulseAudio depends on `dbus` and `ConsoleKit2` after installing pulse you need to enable the corresponding system services.
```
Comment (Gottox) [inline faq/index.markdown]: That's not true, pa does not depend on ConsoleKit.

File: faq/index.markdown [hunk 6]
```diff
@@ -98,6 +98,62 @@ The installer has to be executed as `root` user, if you logged in as `anon` you
...
+
+```
+# xbps-install -S alsa-utils pulseaudio ConsoleKit2
+# ln -s /etc/sv/dbus /var/service/
+# ln -s /etc/sv/cgmanager /var/service/
+# ln -s /etc/sv/consolekit /var/service/
+```
+
+To get PulseAudio working correctly you need to make sure that your session is started with a ConsoleKit seat and a dbus session bus. Some display managers do that for you, in other cases where you just start a Xorg session with `startx` you have to start them from `.xinitrc`.
+
```
Comment (Gottox) [inline faq/index.markdown]: I'm running pa and elogind. consolekit isn't even installed. I'm actually not sure if a loginmanager is needed at all if the permissions are correct.

Comment (Duncaen) [inline faq/index.markdown]: Ok, I'm going to add a Xorg section where I write something about elogind and consolekit.
From the pulse section I just mention the three methods and link to the Xorg session for elogind/consolekit setups.

**Pull request merged**